lambdas/shared/t4_lambda_shared/utils.py

```diff
@@ -26,3 +28,12 @@ def make_json_response(status_code, json_object, extra_headers=None):
         headers.update(extra_headers)
 
     return status_code, json.dumps(json_object), headers
+
+
+def read_body(resp):
```

Comment:
perfect can you a) docstring; b) have dima's code in /preview test use this same function?

Reply:
Yep! Didn't know if you would want me to pull that in or not in this PR but will do.